rust/psibase/src/to_key.rs

```diff
@@ -45,6 +45,21 @@ pub trait ToKey {
     }
 }
 
+/// A serialized key (non-owning)
```

Comment:
`Vec<u8>` is owning

Reply:
Good point about this. I renamed to `RawKey` to avoid confusion with C++ side and kept it owned.